schema.sql

```diff
@@ -74,4 +74,10 @@ CREATE TABLE visits (
    FOREIGN KEY(vets_id)
     REFERENCES vets(id)
      ON UPDATE CASCADE
-);
\ No newline at end of file
+);
+
```

Comment:
- [ ] Kindly add indexes inside the schema file, it will be like this:
```SQL
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```